src/components/SelectOption/SelectOption.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import usersFromServer from '../../api/users';
+
+interface Props {
+  name: string;
+  value?: string;
+  label: string;
+  placeholder: string;
+  hasError?: boolean;
+  onSelectedUser: React.Dispatch<React.SetStateAction<number>>;
```

Comment:
easier, right? use it in the future
```suggestion
  onSelectedUser: (currentUserId: number) => void;
```